README.md

```diff
@@ -15,14 +15,25 @@ const path = require('path');
 const nconf = require('nconf');
 require('./lib/nconfetti');
 
-const dir = path.join(__dirname, 'tests', 'configs', 'env');
+const path = path.join(__dirname, 'tests', 'configs', 'env');
```

Comment:
wouldn't this be easier to read with `path.resolve(__dirname, 'tests/configs/env');`?